src/roadmap/releases.md

```diff
@@ -235,10 +235,31 @@ release is predetermined as the value of `$WORKFLOW_RELEASE`.
 
 ### Step 7: Assemble Master Changelog
 
-Each component already updated its release notes on GitHub with CHANGELOG content. The
-bodies of each component's release notes should be concatenated into a single gist. Note that there
-may be more than one release per component--and more than one set of release notes--included in the
-Workflow release.
+Each component already updated its release notes on GitHub with CHANGELOG content. We'll now
+generate the master changelog for the Workflow chart, consisting of all aforementioned component changes
+as well as those non-component repo changes needing to be manually added.
+
+We'll employ the same `generate_params.toml` and `components.json` files as used in Step 4 above, this time
+invoking `deisrel changelog global` to get all component changes between the tag existing in the `WORKFLOW_PREV_RELEASE`
+chart and the _most recent_ release tag existing in GitHub. (Therefore, if there are any unreleased commits in
+a component repo, they will not appear here):
+
+```bash
+deisrel changelog global $HOME/.helmc/workspace/charts/workflow-$WORKFLOW_PREV_RELEASE/tpl/generate_params.toml \
+  components.json > $WORKFLOW_RELEASE
+```
+
+To get non-component repo changelogs (presumably tagged in Step 3 above), one can issue a command like the following
```

Comment:
If you do it this way, the potential is there to pull in commits that were merged *after* the tag we care about. Whereas scraping them from the actual GH release notes would always be the right content. In most cases they will agree since we do this release dance in a short period of time, but getting them from the release itself is going to be safer.